recipes/pytables/meta.yaml

```diff
@@ -0,0 +1,77 @@
+{% set version="3.2.3.1" %}
+
+package:
+    name: pytables
+    version: {{ version }}
+
+source:
+    fn: PyTables-v.{{ version }}.tar.gz
+    url: https://github.com/PyTables/PyTables/archive/v.{{ version }}.tar.gz
+    sha256: e422a57f0507d03c4bdbf1629b6abcd1faa83338cd2fce4747e8390d3290c556
+    patches:
+        - setup.patch
+        - win-c.patch  # [win]
+        - tables_init.patch  # [win]
+
+build:
+    number: 0
+    skip: True  # [win]
```

Comment:
The history here is to skip Windows, get this in a `feedstock`, iterate there to fix Windows leaving `staged-recipes` CIs free.

However, that is not the main issue with this PR. #1133 is :unamused: